cli/internal/helm/client.go

```diff
@@ -215,6 +215,29 @@ func (c *Client) updateOperatorCRDs(ctx context.Context, chart *chart.Chart) err
 	return nil
 }
 
+// isUpgrade returns true if the new version is greater than the current version.
+// Versions should adhere to the semver spec, but this function will prefix the versions with 'v' if they don't.
+func isUpgrade(currentVersion, newVersion string) bool {
+	if !strings.HasPrefix(currentVersion, "v") {
+		currentVersion = "v" + currentVersion
+	}
+	if !strings.HasPrefix(newVersion, "v") {
+		newVersion = "v" + newVersion
+	}
+
+	// If the current version is not a valid semver,
+	// we cant compare it to the new version.
+	// -> We can't upgrade.
+	if !semver.IsValid(currentVersion) {
+		return false
```

Comment:
Maybe treat this as an error so it's easier to spot?

Reply:
I agree. Adjusted the debug messages to be a bit clearer.

Ideally I would like the user to see what exactly is happening when an upgrade is performed, e.g. what updates are actually applied. 
But I guess that will be implemented once we have the complete Constellation updates ready to go (as its already defined in the rfc).
Imo we should not hack in any extra logging (apart from debug messages) at the current time, and leave this open for the future.